src/locales/en/about_command.json

```diff
@@ -0,0 +1,25 @@
+{
+  "command_info": {
+    "description": "Get information about XP.",
+    "name": "about"
+  },
+  "title": {
+    "title": "Reimagine your Community"
+  },
+  "fields": {
+    "summary_title": "Top-Tier Leveling Solution",
+    "summary_description": "Elevate your Community to the next Level with Top-Tier Leveling, endless Customisability and more.",
+
+    "server_support_title": "Join our Server",
+    "fields.server_support_link": "[Join XP]({{link}})",
+
+    "vote_title": "Vote for XP!",
+    "vote_link": "[top.gg]({{link}})",
+
+    "status_title": "Service Status",
+    "status_link": "[Status]({{link}})"
+  },
+  "footer": {
+    "response_shard_company_info": "Response time: {{response}}ms • Shard: {{shard}} • © {{company}}"
```

Comment:
These symbols are a headache to translate.
Please split them individual keys.
F.e. `response_time_n`, `shard_n`, ...